	* gio/Makefile.in: Disable build in subdirectory tests, which does
	not build out of the box on Solaris

*** gio/Makefile.in.original	Tue Feb 16 17:28:32 2010
--- gio/Makefile.in	Tue Feb 16 17:28:56 2010
*************** ETAGS = etags
*** 258,264 ****
  CTAGS = ctags
  am__tty_colors = \
  red=; grn=; lgn=; blu=; std=
! DIST_SUBDIRS = libasyncns xdgmime inotify fen win32 . fam tests
  DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
  am__relativize = \
    dir0=`pwd`; \
--- 258,264 ----
  CTAGS = ctags
  am__tty_colors = \
  red=; grn=; lgn=; blu=; std=
! DIST_SUBDIRS = libasyncns xdgmime inotify fen win32 . fam 
  DISTFILES = $(DIST_COMMON) $(DIST_SOURCES) $(TEXINFOS) $(EXTRA_DIST)
  am__relativize = \
    dir0=`pwd`; \
***************
*** 602,608 ****
  TEST_PROGS = 
  NULL = 
  SUBDIRS = $(am__append_1) $(am__append_3) $(am__append_7) \
! 	$(am__append_10) . $(am__append_13) tests
  @MS_LIB_AVAILABLE_TRUE@@OS_WIN32_AND_DLL_COMPILATION_TRUE@noinst_DATA = gio-2.0.lib
  @MS_LIB_AVAILABLE_TRUE@@OS_WIN32_AND_DLL_COMPILATION_TRUE@install_ms_lib_cmd = $(INSTALL) gio-2.0.lib $(DESTDIR)$(libdir)
  @MS_LIB_AVAILABLE_TRUE@@OS_WIN32_AND_DLL_COMPILATION_TRUE@uninstall_ms_lib_cmd = -rm $(DESTDIR)$(libdir)/gio-2.0.lib
--- 602,608 ----
  TEST_PROGS = 
  NULL = 
  SUBDIRS = $(am__append_1) $(am__append_3) $(am__append_7) \
! 	$(am__append_10) . $(am__append_13) 
  @MS_LIB_AVAILABLE_TRUE@@OS_WIN32_AND_DLL_COMPILATION_TRUE@noinst_DATA = gio-2.0.lib
  @MS_LIB_AVAILABLE_TRUE@@OS_WIN32_AND_DLL_COMPILATION_TRUE@install_ms_lib_cmd = $(INSTALL) gio-2.0.lib $(DESTDIR)$(libdir)
  @MS_LIB_AVAILABLE_TRUE@@OS_WIN32_AND_DLL_COMPILATION_TRUE@uninstall_ms_lib_cmd = -rm $(DESTDIR)$(libdir)/gio-2.0.lib
